src/shared/firewall-util-nft.c

```diff
@@ -728,6 +748,10 @@ int fw_nftables_init(FirewallContext *ctx) {
         if (r < 0)
                 return r;
 
+        r = fw_nftables_init_family(nfnl, AF_INET6);
+        if (r < 0)
+                log_notice_errno(r, "kernel does not support ipv6 NAT");
```

Comment:
this is a "library" call, and shouldn't log beyond LOG_DEBUG log level thus, also, please include `: %m` in the log message

Reply:
right, changed to _debug+%m.